templates/includes/page_metadata.html

```diff
@@ -0,0 +1,16 @@
+<meta name="description" content="{{ page_description }}" />
+<meta name="keywords" content="{{ page_keywords }}" />
+<title>{{ site_title }} : {{ page_title }}</title>
+
+<meta property="og:type" content="website" />
+<meta property="og:site_name" content="{{ site_title }}" />
+<meta property="og:title" content="{{ site_title }} : {{ page_title }}" />
+<meta property="og:description" content="{{ page_description }}" />
+<meta property="og:url" content="{{ page_url }}" />
+<meta property="og:image" content="{{ base_url }}{{ static_url }}img/logo_squares.png" />
```

Comment:
```suggestion
<meta property="og:image" content="{{ base_url }}{{ static_url }}img/logo_squares.png" />
<meta property="og:image:alt" content="Django Packages logo" />
<meta property="og:image:width" content="195" />
<meta property="og:image:height" content="93" />
<meta property="og:image:type" content="image/png" />
```
Some open graph tags specific for the image we added also in djangoproject.com to let card generator create previews before download the image file.

See for example: `djangoproject/templates/base.html`

P.S. the image file you defined seems too small for all social/chat preview generator.

Reply:
> P.S. the image file you choosed seems too small (w x h) for all social/chat preview generator.

I was not able to find a larger size of the current logo in the repository so I added this one. I have created a banner on Figma that uses the logo.

![dpg](https://user-images.githubusercontent.com/24854406/204089194-89ade268-a9ab-4199-bf00-26cd0d871599.png)

Figma: https://www.figma.com/file/auNUlYXMqjBRR9QYHQWI1Q/Untitled?node-id=5%3A17&t=hTKcmqVLgfRtpbYg-1

Maybe we can use this or we can add a larger size of the logo. C.C: @jefftriplett @ryancheley

Reply:
@jefftriplett I’m not sure if there is already a larger logo sitting around somewhere. If not, this logo work by @saadmk11 is really good

Reply:
The best practice for Open Graph images is to have a wide image with an aspect ratio of 1.9:1 or 2:1 and nowadays at least a width of 1200px. So if you're going to create a new image from scratch to use in `og:image` I suggest to create one with 1200x630 or 1200x600 pixels dimensions.

Reply:
### Summary Card

> Images for this Card support an aspect ratio of 1:1 with minimum dimensions of 144x144 or maximum of 4096x4096 pixels. Images must be less than 5MB in size. The image will be cropped to a square on all platforms. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

\-- from twitter: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary

### Large Summary Card
We can also checkout out large summary card if we want to provide more information.

> Images for this Card support an aspect ratio of 2:1 with minimum dimensions of 300x157 or maximum of 4096x4096 pixels. Images must be less than 5MB in size. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

\-- from twitter: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image

Reply:
> The best practice for Open Graph images is to have a wide image with an aspect ratio of 1.9:1 or 2:1 and nowadays at least a width of 1200px. So if you're going to create a new image from scratch to use in `og:image` I suggest to create one with 1200x630 or 1200x600 pixels dimensions.

Thanks, this will be useful to we if decide to add a custom image

Reply:
This is how it looks like:
![Screenshot from 2022-11-26 21-06-07](https://user-images.githubusercontent.com/24854406/204095545-cb31e26e-afd0-45a0-9b5a-501eda45ba84.png)

The image uses `1200x600` following @pauloxnet's suggestion.

Reply:
These look great to me. I believe the only original art we have are in the repo or we don't have access to it anymore. I'm happy to create a resources repo and share access so we can keep that out of the repo (or we can bundle it here and I can dockerignore it).

Reply:
@jefftriplett Looks like the repository already has a directory [design/](https://github.com/djangopackages/djangopackages/tree/main/design) to store design files. So, I have added the open graph banner `svg` to that directory for now. We can probably `.dockerignore` the `design/` directory which already contains old design files (e.g: svg, psd, pdf).